requirements/requirements.txt

```diff
@@ -28,16 +28,11 @@ neon-utterance-normalizer-plugin~=0.1
 # TODO: Patching test failures https://github.com/NeonGeckoCom/NeonCore/actions/runs/4888865255/jobs/8727164992?pr=422
 requests < 2.30.0
 
-# TODO: Pinning latest stable release
-ovos-workshop==0.0.12
```

Comment:
recommend 0.0.15

Reply:
Not actually a dependency here, it will be updated in neon-utils though